taichi/backends/dx/dx_api.cpp

```diff
@@ -0,0 +1,320 @@
+#include "taichi/backends/dx/dx_api.h"
+
+#include <memory>
+
+#include "taichi/backends/dx/dx_device.h"
+
+TLANG_NAMESPACE_BEGIN
```

Comment:
let also just use explicit namespaces instead of this macro ;)

Reply:
Done.